fluent.runtime/fluent/runtime/fallback.py

```diff
@@ -21,12 +24,12 @@ class FluentLocalization:
 
     def __init__(
         self,
-        locales: List[str],
-        resource_ids: List[str],
+        locales: Sequence[str],
+        resource_ids: Iterable[str],
```

Comment:
Just read the other day that a `str` is also an `Iterable{str]`, we should avoid that type if we really want multiple strings.

Reply:
In a loosely typed language like Python, I'm not sure this can be avoided. There are two options, in my mind: use a union of a bunch of common invariant collections (`Union[List[str], Tuple[str], AbstractSet[str]]`) or add a runtime check to ensure that a string isn't being passed. I really don't like the first option because you're limiting what the user can pass in (a `List[MyStringSubtype]` can't be passed in, for instance), which is why I changed this to `Sequence[str]` and `Iterable[str]`.